src/app/components/languageSelector.js

```diff
@@ -1,43 +1,49 @@
-import React, { useState } from 'react';
-import 'bulma/css/bulma.css';
+import React, { useState } from "react";
+import "bulma/css/bulma.css";
+import "../../App.css";
 
 function LanguageSelector(props) {
-
-    const {languages, languageChange, goHome, part, language} = props;
-        return (
-            <section className="hero is-info is-bold">
-  <div className="hero-body">
-    <div className="container">
-      <div className="columns">
-        <div className="column" onClick={goHome}>
-        <h1 className="title">
-        GoCorona
-      </h1>
-      <h2 className="subtitle">
-        Corona go!!
-      </h2>
-        </div>
-        <div>
-        
-                  
-          {part!=4?<div className="select is-rounded is-pulled-right"><select onChange={languageChange} >
-            {languages.map((lang, i)=>{
-    
-            return (<option key={i} value={lang} selected={language==lang}>{lang}</option>)
-            })}
-  </select></div>:""}
-  
-
+  const { languages, languageChange, goHome, part, language } = props;
+  return (
+    <section className="hero is-info is-bold">
+      <div className="hero-body">
+        <div className="container">
+          <div className="columns">
+            <div className="column">
+              <div onClick={goHome} className="backButton">
```

Comment:
Can we hide the back button on the home page? Because it does not make sense to have a back button which does nothing.